### PR TITLE
Updated my new written documentation, added screenshots and reorganised the table of contents

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by thatman84** [comment]
Just reading through and I would suggest this statement is changed

 # Getting Started: Gamepad setup
 
+RetroArch is intented for playing games
+
 ## Setting up a Gamepad

To 
 # Getting Started: Gamepad setup
 
+RetroArch is intented to be easily controlled with a gamepad.
+
 ## Setting up a Gamepad

**Comment by ghost** [comment]
@thatman84 Good suggestion, I've changed it.

Please let me know if I could improve anywhere else!

**Comment by inactive123** [comment]
Hi,

@RobLoach Can you do a final review of this?

File: docs/guides/download-cores.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Cores are essentially other programs and games that run through RetroArch. Retro
 ## Installing cores through package manager (Ubuntu PPA only)
 
 !!! note
-    Installing RetroArch through the Ubuntu PPA will disable the "Core Updater" option in RetroArch's interface
+    Installing RetroArch through the Ubuntu PPA will disable the "Core Updater" option in RetroArch's interface, Therefore core installation needs to happen through the Ubuntu package manager.
```
Comment (RobLoach) [inline docs/guides/download-cores.md]: Therefore should likely just be `therefore`.

File: docs/guides/launch-content.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+!!! note
+    If the game is zipped you will be presented with two options. *Browse archive* will allow you to open it and select the contents. While *Load archive* will try and load the archive as an image. In most cases if you simply just compressed the image, either option will do.
+
+!!! note
+    RetroArch will automatically choose the appropriate core for your content based on file extension. However if two or more cores use the same file extension, it will ask you to choose the core. You should always choose the platform the game was suppose to run on.
+
+### Playing
+
+Well done, you have finally launched your first game with RetroArch!
```
Comment (RobLoach) [inline docs/guides/launch-content.md]: Don't need the exclamation. See Google's notes on documentation writing with exclamation points: https://developers.google.com/style/exclamation-points

File: mkdocs.yml [hunk 3]
```diff
@@ -4,62 +4,19 @@ repo_url: 'https://github.com/libretro/docs'
...
     - Downloading, Installing and Updating RetroArch:
       - 'Windows': 'guides/install-windows.md'
+      - 'GNU/Linux': 'guides/install-gnu.md'
     - Getting Started:
+      - 'Welcome': 'guides/welcome-getting-started.md'
+      - 'User Interface': 'guides/navigating.md'
       - 'Keyboard Controls': 'guides/retroarch-keyboard-controls.md'
+      - 'Gamepad Setup': 'guides/gamepad-setup.md'
+      - 'Installing Cores': 'guides/download-cores.md'
+      - 'Using file-browser': 'guides/file-browser.md'
```
Comment (RobLoach) [inline mkdocs.yml]: Rather than `Using file-browser`, just have the title case of `File Browser`. The "Using" part is implied.

**Review by RobLoach — CHANGES_REQUESTED** [review]
Looks pretty good! Some minor changes, but looks like a good update.

**Comment by inactive123** [comment]
OK @RobLoach, how are we progressing here?

**Comment by ghost** [comment]
@twinaphex @RobLoach 

Sorry guys I've been super behind. Let me add @RobLoach's suggestions

**Comment by ghost** [comment]
OK that should be all. @RobLoach do you mind taking another look?

**Review by RobLoach — APPROVED** [review]
Looks good! Thanks a lot, looks like some great re-organisation.

**Comment by inactive123** [comment]
Alright, since RobLoach approves of it, I guess we can merge this now.

**Pull request merged**